src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -15,20 +15,26 @@ const WAVEFROM_SCALE_FACTOR = 5
 let recordingLength = 0;
 let recordBuffer = [[], []];
 let isRecording = false;
-let isMonitoring = false;
-let visualizationEnabled = true;
+let initCount = 0;
```

Comment:
Sorry, the update is changed in the next PR. I forget to make sync all of the PR since this issue is reported in the next PR last time. 

I will make everything organized before requesting a new round of review.